### PR TITLE
[Material] Added focusnodes in textfield demo

### PR DESCRIPTION
**Pull request by Alabhya268**
PR for #422 

@rami-a, I've added focusnodes, Please review this PR. 

https://user-images.githubusercontent.com/57143358/107058144-3eeaeb00-67fa-11eb-82b7-cddb247dc7d4.mp4

### CODE REVIEW

File: lib/demos/material/text_field_demo.dart [hunk 1]
```diff
@@ -106,6 +110,28 @@ class _PasswordFieldState extends State<PasswordField> {
...
+  FocusNode _phoneNumber, _email, _lifeStory, _password, _retypePassword;
+
+  @override
+  void initState() {
+    _phoneNumber = FocusNode();
+    _email = FocusNode();
+    _lifeStory = FocusNode();
+    _password = FocusNode();
+    _retypePassword = FocusNode();
+    super.initState();
```
Comment (rami-a) [inline lib/demos/material/text_field_demo.dart:122]: Please call super first for `initState()`, it is fine to call last for `dispose()`

**Review by rami-a — APPROVED** [review]
LGTM except for a small comment!

We're still waiting to resolve our CI checks however, so please be patient until we do and then I can merge this.

**Comment by Alabhya268** [comment]
I've made the necessary changes now.

**Comment by Alabhya268** [comment]
I've made changes needed.

**Comment by Alabhya268** [comment]
@rami-a , Should I also add focusnode in shrine and rally login form, If so should I make separate PR ? Let me know what you think.

**Comment by rami-a** [comment]
> @rami-a , Should I also add focusnode in shrine and rally login form, If so do I do it in a separate PR or in this one? Let me know what you think.

Sure that's a good idea! I'd say make it a separate PR. Thanks!

**Comment by rami-a** [comment]
Hey @Alabhya268, I have now fixed the broken tests. Please update your branch with the latest changes from master when you get a chance and then it should be good to merge!

**Comment by rami-a** [comment]
Looks like after merging your divider PR, this one needs to be updated again

**Pull request merged**